readme.md

```diff
@@ -1,71 +1,71 @@
 
 Where as the models normally provide a way to encapsulate business logic and data, **Views & Controllers** are responsible for telling us what to do and how to display that data.
 
-### Doc Dive (5 / 10)
+### Doc Dive (5 min)
 
 Read Parts 1-3: http://guides.rubyonrails.org/action_controller_overview.html
 
-### Turn & Talk (5 / 15)
+### Turn & Talk (5 min)
 
 With a partner, discuss:
 
 1. What is the role of a controller in a Rails application?
 2. What are the conventions for naming Rails controllers?
-3. What are the benefits following the 'Rails Way'?
+3. What are the benefits of following the 'Rails Way'?
 
-### rMVC: Revisited (10 / 25)
+### rMVC: Revisited (10 min)
 
 The design pattern that rails is built around is rMVC - Router, Model, View and Controller.
 
 ![rMVC](images/mvc_diagram.png)
 
 **Question:** Who can remind us the role the View plays in a Rails application?
 
-Life Cycle of a request/response in Rails:
+Life Cycle of a Request/Response in Rails:
 
-1. A user of our web application submits a request to our application's server. It can come in a myriad of ways. Maybe someone typing in a URL and hitting enter or maybe a user submitting a form on our application.
+1. A user of our web application submits a request to our application's server. It can come in a number of ways - typing in a URL and hitting enter or from submitting a form on our application.
 
 2. The request hits the router of the application.
 
-3. The application than either doesn't recognize the route (error) or it does recognize it(route) and sends it to a controller.
+3. The application will either not recognize the route and error. Or, it will recognize the route and send it to a controller.
 
-4. Once the request hits the controller, its then going to query the database through Active Record(the model) for the information specified in the controller.
+4. Once the request hits the controller, the controller will query the database through the model using Active Record for the information specified in the controller.
 
-5. Once the controller has the information from the model that it needs, it sends it to the view
+5. Once the controller has the information from the model that it needs, it sends it to the view.
 
 6. The view takes the objects from the controller, and sends a response to the user.
 
 Let's dive into an example, and see MVC in action by writing some Rails code.
 
-## Set-Up (5 / 30)
+## Set-up (5 min)
 
 The starter code for this lesson is where we left off after the Models and Migrations class for our Tunr application. We should already have all the Active Record components completed, so no need to worry about changing the database moving forward.
 
-Or if you want to start fresh from a working solution, feel free to begin by cloning down the below repo and running the necessary setup.
+Or, if you want to start fresh from a working solution, feel free to begin by cloning down the below repo and running the necessary setup.
 
```

Comment:
I would actually have them all start with cloning down the fresh working copy. Could save time with errors and bugs this way.